src/interface/update.hpp

```diff
@@ -33,6 +33,23 @@ namespace parthenon {
 
 namespace Update {
 
+// Calculate the flux divergence for a specific component l of a variable v
+KOKKOS_FORCEINLINE_FUNCTION
+Real FluxDiv_(const int l, const int k, const int j, const int i, const int ndim,
```

Comment:
We should consider renaming `FluxDiv_` to `FluxDiv` or `FluxDivHelper` or something if it's publicly exposed.

Reply:
Done.